File: OperatorApp/frontend/src/app/DTOs/BE2FE_ActionForItemDto.ts
export interface BE2FE_ActionForItemDto {
    itemSerialNumber: string;
    itemType: string;
    actionName: string;
    actionSWVersionForExecution: string;
    actionExeName: string;
    latestExecutionDateTimeUTC: string;
    latestResult: string;
    latestOperatorName: string;
    latestActionVersionNumber: string;
  }

File: OperatorApp/Backend/DTOs/FE_BE_DTOs/BE2FE_ActionForItemDto.cs
namespace Backend.DTOs;

public class BE2FE_ActionForItemDto
{
    public string itemSerialNumber { get; set; }
    public string itemType { get; set; }
    public string actionName { get; set; }
    public string actionSWVersionForExecution { get; set; }
    public string actionExeName { get; set; }

    public DateTime? latestExecutionDateTimeUTC { get; set; }
    public string latestResult { get; set; }
    public string latestOperatorName { get; set; }
    public string latestActionVersionNumber { get; set; }
}